constants/currency.ts

```diff
@@ -15,75 +19,7 @@ export const CATEGORY: Category[] = [
 ];
 export const CATEGORY_MAP = keyBy(CATEGORY);
 
-export const SYNTHS = [
-	'sBTC',
-	'sETH',
-	'sXRP',
-	'sBCH',
-	'sLTC',
-	'sEOS',
-	'sBNB',
-	'sXTZ',
-	'sXMR',
-	'sADA',
-	'sLINK',
-	'sTRX',
-	'sDASH',
-	'sAAVE',
-	'sUNI',
-	'sYFI',
-	'sDOT',
-	'sREN',
-	'sCOMP',
-	'sETC',
-	'iBTC',
-	'iETH',
-	'iXRP',
-	'iBCH',
-	'iLTC',
-	'iEOS',
-	'iBNB',
-	'iXTZ',
-	'iXMR',
-	'iADA',
-	'iLINK',
-	'iTRX',
-	'iDASH',
-	'iETC',
-	'sFTSE',
-	'sNIKKEI',
-	'sTSLA',
-	'sMSFT',
-	'sFB',
-	'sAMZN',
-	'sAAPL',
-	'sNFLX',
-	'sGOOG',
-	'sCOIN',
-	'sXAU',
-	'sXAG',
-	'sOIL',
-	'iOIL',
-	'sEUR',
-	'sJPY',
-	'sUSD',
-	'sAUD',
-	'sGBP',
-	'sKRW',
-	'sCHF',
-	'sCEX',
-	'sDEFI',
-	'iCEX',
-	'iDEFI',
-	'iAAVE',
-	'iUNI',
-	'iYFI',
-	'iDOT',
-	'iREN',
-	'iCOMP',
-];
-
-export const SYNTHS_MAP = keyBy(SYNTHS);
+export const SYNTHS = Object.keys(Synths) as CurrencyKey[];
```

Comment:
e.g. would just be `string[]` instead of `CurrencyKey[]` if you want to go that route

Reply:
this has been removed in favor of the `Synths` enum which does basically the same thing.